### ISSUE TITLE
Support addr2 (and addrN in general)

### ISSUE THREAD

**Issue by jluttine** [openstreetmap/Nominatim#1345]
Some buildings may have multiple addresses. One way (or only used way?) to tag these is by `addr2:*`, `addr3:*`, and so on up to `addr9:*`. It seems to me that Nominatim only considers `addr:*` tags. If so, it would be great to add support for `addrN:*` so it doesn't matter which of the addresses a user enters, they'll find the building.

For more information, see: https://wiki.openstreetmap.org/wiki/Proposed_Features/addrN

**Comment by tohaklim** [comment]
Another way to tag these (and one already supported by most if not all data consumers) is to place a node with addr tags in the building outline

**Comment by jluttine** [comment]
@tohaklim Just to make sure I understood correctly: If a building has two different addresses, I choose two different nodes of the building outline and add addr tags to those? If the building has more addresses than nodes, one can always add extra nodes to the building outline path. Or did you mean adding a separate addr node somewhere inside the building area? Is it a problem that in neither case, the addr tagged node has no building tag itself?

**Comment by tohaklim** [comment]
not quite, I was referring to the approach mentioned in the wiki page above: create a free standing node inside the building area (or associate it with the building through other means, but inside the area is most logical), and put addr tags on it. An address is not necessarily a building, so I don't see issues with not having building tags on the extra addr notes

**Comment by c933103** [comment]
If the two addresses belong to/reflect location of two different doors/opening of the same building, them I would tag each of the doors themselves with their respective address.

**Comment by drmx** [comment]
if there is a point with an address inside the contour, then what address will the POI have inside? they can only be found at the address in the outline
